### PR TITLE
Weekly Changelog 2023-04-24 to 2023-05-01

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Additional context
`Automatically generated as a draft. Please copy-edit before merging.`

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -24,6 +26,21 @@ Add the first Exodii mission
 Combat Capable Mounts
 Add new Exodii NPC: Luliya
 Add new weather: mist and fog. maybe.
```
Comment (GuardianDll) [inline data/changelog.txt:28]: While we are here
```suggestion
Add new weathers: mist and fog
```

File: data/changelog.txt [hunk 2]
```diff
@@ -23,7 +25,22 @@ Updates to several existing maps
 Add the first Exodii mission
 Combat Capable Mounts
 Add new Exodii NPC: Luliya
-Add new weather: mist and fog. maybe.
+Add new weather types: mist and fog
+adds .303 British as an ammo type in the game, plus corresponding FMJ, SP, and BTHP rounds, craft recipes, item groups, migrations, and the SMLE Mk III and No. 4 Mk I rifles
```
Comment (Maleclypse) [inline data/changelog.txt:29]:
```suggestion
```

File: data/changelog.txt [hunk 3]
```diff
@@ -23,7 +25,22 @@ Updates to several existing maps
...
+Allow static NPCs to be given randomly generated missions.
+Allow players to keep bees
+Add carpet armor
+Adds a Cheesemaker hobby in character creation
+Added Proficiencies & Weakpoints to Zombie-Adjacent Monsters
+Hospitals can now spawn with piles of corpses outside; minor improvements
```
Comment (Maleclypse) [inline data/changelog.txt:35]:
```suggestion
```

File: data/changelog.txt [hunk 4]
```diff
@@ -23,7 +25,22 @@ Updates to several existing maps
...
+map update: music store
```
Comment (Maleclypse) [inline data/changelog.txt:36]:
```suggestion
```

File: data/changelog.txt [hunk 5]
```diff
@@ -23,7 +25,22 @@ Updates to several existing maps
...
+map update: pharmacy
```
Comment (Maleclypse) [inline data/changelog.txt:37]:
```suggestion
```

File: data/changelog.txt [hunk 6]
```diff
@@ -23,7 +25,22 @@ Updates to several existing maps
...
+Paramilitary Zombies and everyday carry
+Apartment complex: parking garage, lobby variants, roof additions
+Added missing proficiency to appliance crafts & introduced new background with said proficiency
```
Comment (Maleclypse) [inline data/changelog.txt:40]:
```suggestion
```

File: data/changelog.txt [hunk 7]
```diff
@@ -23,7 +25,22 @@ Updates to several existing maps
...
+Profession Requirements
+obsolete the power armor bionics
```
Comment (Maleclypse) [inline data/changelog.txt:42]:
```suggestion
```

File: data/changelog.txt [hunk 8]
```diff
@@ -23,7 +25,22 @@ Updates to several existing maps
...
+Map update: regional airport
```
Comment (Maleclypse) [inline data/changelog.txt:43]:
```suggestion
```

File: data/changelog.txt [hunk 9]
```diff
@@ -67,6 +85,12 @@ Remove flaming eye phantom melee attack that can disrupt aim
 Converts most of the armor and clothing still using the old limb system to the new system with sub-limbs
 partial skill levels contribute to most game tests
 Portal Storm Coherency Pass
+Rebalanced 60-round magazines for exodii rifles
+Stop zomborgs from exploding on death
+less insane coal
```
Comment (Maleclypse) [inline data/changelog.txt:90]:
```suggestion
```

File: data/changelog.txt [hunk 10]
```diff
@@ -67,6 +85,12 @@ Remove flaming eye phantom melee attack that can disrupt aim
...
+hardtack doesn't have the gross food vitamin
```
Comment (Maleclypse) [inline data/changelog.txt:91]:
```suggestion
```

File: data/changelog.txt [hunk 11]
```diff
@@ -67,6 +85,12 @@ Remove flaming eye phantom melee attack that can disrupt aim
...
+Prevented melee from breaking the Phase Immersion Suit
```
Comment (Maleclypse) [inline data/changelog.txt:92]:
```suggestion
```

File: data/changelog.txt [hunk 12]
```diff
@@ -67,6 +85,12 @@ Remove flaming eye phantom melee attack that can disrupt aim
...
+Make the following locations more rare: Hazardous waste sarcophagus, military outpost, military helipad, military bunker, LMOE shelter, convention center, bastion fort, cathedral
```
Comment (Maleclypse) [inline data/changelog.txt:93]:
```suggestion
```

File: data/changelog.txt [hunk 13]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
 Prevent fire damaging unbreakable items
 Fix energy guns(AFS) on NPC
 Faction camps now distribute calories based on actual calories and not default calories
+Reduce maid costume and other drops
```
Comment (Maleclypse) [inline data/changelog.txt:125]:
```suggestion
```

File: data/changelog.txt [hunk 14]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+fix Free Merchant guard gear groups
```
Comment (Maleclypse) [inline data/changelog.txt:126]:
```suggestion
```

File: data/changelog.txt [hunk 15]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+fixes #65092, straitens out the Kel-Tec SUB 2000’s folding properties.
```
Comment (Maleclypse) [inline data/changelog.txt:127]:
```suggestion
```

File: data/changelog.txt [hunk 16]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+spawn forest mammals in winter
```
Comment (Maleclypse) [inline data/changelog.txt:128]:
```suggestion
```

File: data/changelog.txt [hunk 17]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+Fix archery stamina use calculations
```
Comment (Maleclypse) [inline data/changelog.txt:129]:
```suggestion
```

File: data/changelog.txt [hunk 18]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+Fixed large wooden box description.
```
Comment (Maleclypse) [inline data/changelog.txt:130]:
```suggestion
```

File: data/changelog.txt [hunk 19]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+Inherit relative rot when disassembling
+Fix UPS powered guns with non integer kJ energy consumption
```
Comment (Maleclypse) [inline data/changelog.txt:132]:
```suggestion
```

File: data/changelog.txt [hunk 20]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+Fixed display of tools required to repair an item
```
Comment (Maleclypse) [inline data/changelog.txt:133]:
```suggestion
```

File: data/changelog.txt [hunk 21]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+correct artifacts spawn in distorted labs
```
Comment (Maleclypse) [inline data/changelog.txt:134]:
```suggestion
```

File: data/changelog.txt [hunk 22]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+resolve issues with telefrag
```
Comment (Maleclypse) [inline data/changelog.txt:135]:
```suggestion
```

File: data/changelog.txt [hunk 23]
```diff
@@ -98,6 +122,21 @@ Ferals can use their guns in GG
...
+Separates weather tolerance traits from stomach related ones
+Fix calculation for inserting into nested containers
+fix random missions
+NPC spawn with reasonable stored_kcal
```
Comment (Maleclypse) [inline data/changelog.txt:139]:
```suggestion
```

File: data/changelog.txt [hunk 24]
```diff
@@ -112,6 +151,9 @@ Refactor some crafting infrastructure to support removal of charges
 Unify gun battery/ups/bionic energy consumption
 Modernize string_formatter
 Migrate some JSON APIs to string_view
+Brief description
```
Comment (Maleclypse) [inline data/changelog.txt:154]:
```suggestion
```

**Review by Maleclypse — COMMENTED** [review]
If someone else wants to review and confirm my changes I'd appreciate it.

File: data/changelog.txt [hunk 25]
```diff
@@ -23,7 +25,15 @@ Updates to several existing maps
...
-Add new weather: mist and fog. maybe.
+Add new weather types: mist and fog
+Allow static NPCs to be given randomly generated missions.
+Allow players to keep bees
+Add carpet armor
+Adds a Cheesemaker hobby in character creation
+Added Proficiencies & Weakpoints to Zombie-Adjacent Monsters
+Paramilitary Zombies and everyday carry
+Apartment complex: parking garage, lobby variants, roof additions
+Profession Requirements
```
Comment (dseguin) [inline data/changelog.txt:36]:
```suggestion
Add new weather types: mist and fog
Allow players to keep bees
Apartment complex: parking garage, lobby variants, roof additions
```

File: data/changelog.txt [hunk 26]
```diff
@@ -11,6 +11,8 @@ Added sorting, showing&hiding pocket contents, and scrolling via mouse in invent
 New game event that triggers on load or new game start
 Add option to hide bionics
 Allow JSON monster special attacks to use the dialog condition system
+Quadrupedal movement with paws
+Confirmation before attacking neutral mobs
```
Comment (dseguin) [inline data/changelog.txt:15]:
```suggestion
Confirmation before attacking neutral mobs
```

File: data/changelog.txt [hunk 27]
```diff
@@ -54,6 +64,7 @@ Mouse thumb button support
 [Xedra Evolved] Museum Location for Xedra Evolved
 [Magiclysm] Adds several new professions to the Magiclysm mod
 [Bombastic Perks] Added resurrecting meat monstrosities to bombastic perks
+Forcefield and Evasion enchants
```
Comment (dseguin) [inline data/changelog.txt:67]:
```suggestion
[Bombastic Perks] Forcefield and Evasion enchants
```

File: data/changelog.txt [hunk 28]
```diff
@@ -67,6 +78,8 @@ Remove flaming eye phantom melee attack that can disrupt aim
 Converts most of the armor and clothing still using the old limb system to the new system with sub-limbs
 partial skill levels contribute to most game tests
 Portal Storm Coherency Pass
+Rebalanced 60-round magazines for exodii rifles
+Stop zomborgs from exploding on death
```
Comment (dseguin) [inline data/changelog.txt:82]:
```suggestion
Stop zomborgs from exploding on death
```

File: data/changelog.txt [hunk 29]
```diff
@@ -98,6 +111,10 @@ Ferals can use their guns in GG
 Prevent fire damaging unbreakable items
 Fix energy guns(AFS) on NPC
 Faction camps now distribute calories based on actual calories and not default calories
+Inherit relative rot when disassembling
+Separates weather tolerance traits from stomach related ones
+Fix calculation for inserting into nested containers
+fix random missions
```
Comment (dseguin) [inline data/changelog.txt:117]:
```suggestion
Fix calculation for inserting into nested containers
```

File: data/changelog.txt [hunk 30]
```diff
@@ -112,6 +129,8 @@ Refactor some crafting infrastructure to support removal of charges
 Unify gun battery/ups/bionic energy consumption
 Modernize string_formatter
 Migrate some JSON APIs to string_view
+Add clang-tidy check looking for opportunities to convert functions to use string_view
+Add units::temperature_delta
```
Comment (dseguin) [inline data/changelog.txt:133]:
```suggestion
Add units::temperature_delta
```

**Pull request merged**